lambda_backend/route_microservice/src/common/types.ts

```diff
@@ -54,14 +54,14 @@ export interface GetRouteDetailsEvent {
   };
 }
 
-export interface UpvoteRouteEvent extends AuthHeader {
+export interface ToggleUpvoteRouteEvent extends AuthHeader {
   body: {
     username: string;
     createdAt: string;
   };
 }
 
-export type ReportRouteEvent = UpvoteRouteEvent;
+export type ReportRouteEvent = ToggleUpvoteRouteEvent;
```

Comment:
Is this some temp thing?

Reply:
In case we change one of them in the future. The naming is clear